examples/example1.jl

```diff
@@ -16,15 +17,17 @@ add!(m, Commodity(:PK))
 
 add!(m, Consumer(:RA, benchmark=150.))
 
-add!(m, Production(:X, 1, :PX, 100, [Input(:PL, 50), Input(:PK, 50)]))
-add!(m, Production(:Y, 1, :PY, 50, [Input(:PL, 20), Input(:PK, 30)]))
-add!(m, Production(:U, 1, :PU, 150, [Input(:PX, 100), Input(:PY, 50)]))
+add!(m, Production(:X, 1, :PX, 100, [Input(:PL, :(50 * $inputcoeff)), Input(:PK, 50.)]))
+add!(m, Production(:Y, 1, :PY, 50, [Input(:PL, 20.), Input(:PK, 30.)]))
+add!(m, Production(:U, 1, :PU, 150, [Input(:PX, 100.), Input(:PY, 50.)]))
 
 add!(m, Demand(:RA, :PU, [Endowment(:PL, :(70 * $endow)), Endowment(:PK, 80.)]))
 
 solve!(m, cumulative_iteration_limit=0)
 
 set_value(endow, 1.1)
+set_value(inputcoeff, 1.2)
+
```

Comment:
I think ideally we would actually not change this example in the PR, but instead add another test case where you have a parameter in an input. Two benefits: 1) we leave this example to match the original, and 2) tests run routinely.